demo/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "gatsby build",
-    "develop": "gatsby develop -p 8001",
+    "develop": "GATSBY_NEWRELIC_ENV=development gatsby develop -p 8001",
```

Comment:
what does this flag do?

Reply:
When you boot up the demo site, you'll see a warning: 

```
GATSBY_NEWRELIC_ENV env variable is not set
```

This comes from the `gatsby-plugin-newrelic` plugin. This quiets that warning 😄

Reply:
ahhhh I see :) thank you